dev/Tiltfile

```diff
@@ -177,7 +180,7 @@ local_resource(
             port = 4012,
         ),
     ),
-    deps = _buck2_dep_inputs(api_target),
+    deps = _buck2_dep_inputs(api_target_ci),
```

Comment:
don't see why this is required? The dev target should also know its files.

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/3572/commits/15424efdab70847b6b0b7a298dc0d97c088f40d8